### ISSUE TITLE
darkwolf 5.2 User Account display with many users

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/themes#88]
@SteppenWolf68 

When there are too many users for one page of the User Account tab, the pagination area doesn't seem to fit with the rest of the theme:

<img width="728" alt="image" src="https://user-images.githubusercontent.com/1443710/197396242-e7fe2d2c-12af-471b-981f-73cee433255b.png">

**Comment by SteppenWolf68** [comment]
Hi @ibennetch a pleasure to greet and thanks for the feedback.

I hope it is solved, I upload the css files to replace, I will appreciate any comment, greetings.

[css.zip](https://github.com/phpmyadmin/themes/files/9902852/css.zip)

**Comment by williamdes** [comment]
Hi @SteppenWolf68 
I uploaded them into af986e40e92f02c8fbf251ffe21aec238ee7eb2e

**Comment by williamdes** [comment]
If you have vs code I advise you to install [Trailing white spaces](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) to have files without training white spaces :)

**Comment by williamdes** [comment]
Hi @SteppenWolf68 
I fixed your css file with bf2484761472f2687def224427de1ccec4ad243c

```
CssSyntaxError: themes/darkwolf/css/theme.rtl.css:1565:27: Missed semicolon

  1563 | }
  1564 | .table > thead {
> 1565 |     vertical-align: bottom --bs-table-bg: #1B1E21;
       |                           ^
  1566 | }
  1567 | .table > :not(:first-child) {
```

**Comment by williamdes** [comment]
Also df7ce470923f0c1a3e6ae024b102afd5b07e6e74
